data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+				decline
+
+	on visit
+		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`
+
+
+
+mission "Adelita 3"
+	name "Film slave transaction"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
+	landing
+	cargo "stealth drones" 1
+	passengers 1
+	infiltrating
+	"apparent payment" 200000
+	source "Maker"
+	destination "Hippocrates"
+	to offer
+		has "Adelita 2: done"
+
+	on offer
+		conversation
+			`You drop off Ale and River at the spaceport, watching as they try to seem neither visibly embarrassed to be in their security uniforms nor uncomfortable with their disguises, even though they're now virtually indistinguishable from other Syndicate personnel; Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+
+
+mission "Adelita 4"
+	name "Take the film to <planet>"
+	description "Take the film to <destination>, where the rest of the Adelita group remained behind. (You will be paid <payment> after you complete the entire job.)"
+	landing
+	passengers 1
+	cargo "stealth drones" 1
+	infiltrating
+	"apparent payment" 200000
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Adelita 3: done"
+
+	on offer
+		conversation
+			branch "player has a fleet"
+				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1
+
+			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but you are low enough that Bunker releases the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
+				goto "on site"
+
+			label "player has a fleet"
+			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```

Comment:
```suggestion
			`It's not hard to find a remote canyon within range of the target area. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape. You are low enough that Bunker can release the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
				goto "on site"

			label "player has a fleet"
			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
```

Shields need to be off if we're going to be expected to believe that the Quicksilvers can kill you later.